packages/dashboard/package.json

```diff
@@ -32,11 +28,20 @@
     "@types/enzyme": "3.1.5",
     "@types/enzyme-adapter-react-16": "1.0.1",
     "@types/jest": "23.0.0",
+    "@types/react": "16.3.16",
```

Comment:
per https://basarat.gitbooks.io/typescript/content/docs/errors/main.html we should move this into peerDependencies as well.

Reply:
That's right, though when I looked inside all other packages in fabric, types/react is under devDependencies. I think the problem right now is that types/react is under dependencies, so it created two copies for you. I've moved it to devDependencies.

Reply:
Are you sitting the duplicate react.d.ts error in a downstream repo? `@types/react` being in devDependencies is absolutely the right fix.

Reply:
yes I was hitting the issue in my own app project which is how I got the issue to Samiya. DevDependencies was my initial thought as well but I wasn't sure after reading the docs. As long as dev dependencies for other modules aren't installed with npm install on the root we should be good here. The docs are not entirely clear to me on this point so I'm hoping that's the case.

Reply:
Ok so I think what will happen is by default we'd be ok leaving in dev dependencies. However, if someone takes a dependency on this and does npm install --dev they would run into the same issue I hit if they have a different version listed in their deps. I think peer deps will avoid this in all cases. Thoughts?

Reply:
No.. npm install --dev will not install the dev dependencies of a package's dependencies. It will only install the direct dev dependencies.

Reply:
Think of it this way. A package's dev dependencies are the packages that said package needs to undergo the "dev"/"build"/"test" cycle that generates the final output. The output, once generated, no longer needs anything that is a devDependency. 

In other words, when you consume said package in a downstream repo, you do not care about all the tools that were involved in transforming the source of the package into the final output (usually, most packages do not publish source) because you don't consume the raw source - you consume only the output. You do not want to install all the build tools e.g. python, various SDKs, gulp or what have you that are not relevant to your own downstream project. This is why NPM never installs a dependency's dev dependencies, even in `npm install --dev`

Reply:
ok great then I'm good. That was the main thing. I think NPM could do a better job clarifying these things in their docs. Seems I'm not the only person who is hazy here. Appreciate the clarify Cliff!

Reply:
p.s. I stand corrected. Historically, there used to be an actual `npm install --dev` that might have indeed exhibited the behavior you spoke about, but if it recursively installed dev dependencies, it was viewed as a bug, even by NPM, and this command has been deprecated.

https://github.com/npm/npm/issues/5554#issuecomment-56121953

Reply:
ok good to know that won't be an issue! Thanks for the link too.